packages/types/tools/copy-ast-spec.ts

```diff
@@ -1,11 +1,39 @@
-import chlidProcess from 'child_process';
+import childProcess from 'child_process';
 import fs from 'fs';
 import path from 'path';
 import { promisify } from 'util';
 
 const readFile = promisify(fs.readFile);
 const writeFile = promisify(fs.writeFile);
-const execAsync = promisify(chlidProcess.exec);
+
+// the promisify util will eat the stderr logs
+async function execAsync(
+  command: string,
+  args: ReadonlyArray<string>,
+  options: childProcess.SpawnOptions,
+): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const child = childProcess.spawn(command, args, {
+      ...options,
+      stdio: 'inherit',
+    });
+
+    child.on('error', e => reject(e));
+    child.on('exit', () => resolve());
+    child.on('close', () => resolve());
+
+    // (error, stdout, stderr) => {
+    //   if (error) {
+    //     console.error(stderr);
+    //     reject(error);
+    //     return;
+    //   }
+
+    //   // console.log(stdout);
+    //   resolve();
+    // },
```

Comment:
Delete?

Reply:
One minor thing if you're going to update this util  anyway to delete the code - one thing that can be nasty to troubleshoot if it ever comes up is the stingy `maxBuffer` size that node gives you by default.

`execa`, for example, ups this to 100MB https://github.com/sindresorhus/execa/blob/fa9cc053837a3c78f61ee1f330aa4048bb8d278f/index.js#L16 so we might want to do something similar

Reply:
In the case of `child_process.exec` - it is built to execute the command, buffer the entire output, then return it as a string.
This is why a buffer is important for it - because it buffers the output and then returns it.

For this I've used `spawn` with `stdio: 'inherit'`.
This means that the log lines will just be streamed directly to the main process's stdout and thus there's no buffer collecting the output for an extended period.
In fact, `spawn` has no option to control the buffer size!